variables.tf

```diff
@@ -148,3 +149,11 @@ variable "cluster_version" {
   description = "The cluster version"
   default     = ""
 }
+
+variable "PostgresConfigmap" {
```

Comment:
The variable name should follow the format convention of all the other variables - postgres_configmap